docs/insomnia/requests.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: article-detail
+title:  Requests
+category: "Requests and Responses"
+category-url: requests-and-responses
+---
+
+Insomnia supports sending requests via [HTTP method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods), [gRPC](https://grpc.io/docs), and [GraphQL](https://graphql.org).
```

Comment:
I won't comment everywhere it's repeated, but should this just be "requests via HTTP", and "HTTP request", instead of "HTTP _method_"? HTTP Method is just a specific part of the HTTP protocol. 

I'm not certain what the wording should be here but it seemed odd at first glance

Reply:
That makes sense, however, for the sentence structure to make sense (I'm listing HTTP, gRPC, and GraphQL), I opted for this.